### ISSUE TITLE
Create templates for other types of articles

### ISSUE THREAD

**Issue by mrshirts** [livecomsjournal/article_templates#19]
However, this should wait until we've nailed down the commonalities between all, so we don't have too many differences between .tex files that all require editing.

**Comment by davidlmobley** [comment]
Waiting on #10 , covered by #20 ; since it's part of my checklist there we don't need to preserve this as a separate issue.
